scripts/build.sh

```diff
@@ -77,6 +77,8 @@ install() {
     inst_multiple -o \
       "$systemdsystemunitdir"/mount-sysroot.service \
       "$systemdsystemunitdir"/sysinit.target.wants/mount-sysroot.service
+
+    inst_simple "/usr/local/etc/initoverlayfs.conf" "/etc/initoverlayfs.conf"
```

Comment:
Have no problem making this /usr/local/etc everywhere if it works, even in initrd

Reply:
Sure, I am doing some tests here, I will update here  as soon I get something working.